armData.json

```diff
@@ -19058,6 +19058,86 @@
         "imageURL": "./imgs/1040904300.png",
         "en": "[4★] Ameno Habakiri"
     },
+    "タミン・サリ": {
+        "ja": "タミン・サリ",
+        "element": "light",
+        "type": "dagger",
+        "skill1": "normalL",
+        "skill2": "non",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 38,
+        "minattack": 410,
+        "hp": 259,
```

Comment:
>"hp": 259,

235 is correct.
imege from internet.
![image](https://user-images.githubusercontent.com/22173829/69327510-01e81680-0c91-11ea-9038-7642c37b0ec4.png)